cmd/start.go

```diff
@@ -38,12 +38,23 @@ var StartCmd = &cli.Command{
 	Name:  "start",
 	Usage: "Start the storage node daemon.",
 	Flags: []cli.Flag{
-		KeyFileFlag,
 		CurioURLFlag,
+		ProofSetFlag,
+		&cli.PathFlag{
+			Name:      "key-file",
+			Usage:     "Path to a file containing ed25519 private key, typically created by the id gen command.",
+			EnvVars:   []string{"STORAGE_PRIVATE_KEY"},
+			TakesFile: true,
+		},
+		&cli.StringFlag{
+			Name:    "config",
+			Usage:   "Path to configuration file.",
+			EnvVars: []string{"STORAGE_CONFIG"},
+		},
```

Comment:
should this one be a `cli.PathFlag` too? Also, I don't know what it does, but should this flag also have `TakesFile: true`?

Reply:
Yeah, probably best to include that in the flags definition. ufave/cli docs are a bit sparse on what this option does, my gut says its related to shell completion: https://github.com/urfave/cli/blob/8f20a3568191ab2a77de5bf37244ad607098c301/flag_impl.go#L55

Will change.